stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.29.2
+version: 1.29.4
```

Comment:
You're adding in a new feature, please bump the minor version of the chart.

Reply:
@timstoop Done